### PR TITLE
Added SRF organization with rights to deploy rights to tt02.

### PR DESCRIPTION
**Pull request by RonnyB71**

### CODE REVIEW

**Comment by RonnyB71** [comment]
@altinnadmin could you take a look at this?

File: orgs/altinn-orgs.json [hunk 1]
```diff
@@ -226,6 +226,19 @@
                 "production"
             ]
         },
+        "srf": {
+            "name": {
+                "en": "Norwegian Civil Affairs Autorithy",
```
Comment (altinnadmin) [inline orgs/altinn-orgs.json:231]: typo :)

**Review by altinnadmin — CHANGES_REQUESTED** [review]
One small typo to fix :)

**Review by altinnadmin — APPROVED** [review]
:shipit:

**Pull request merged**